### PR TITLE
Add GH2 template

### PR DESCRIPTION
**Pull request by cwensley**

### CODE REVIEW

**Comment by benjaminfortunato** [comment]
I'm getting the error message Extract: extension/package.json not found inside zip. I'm using visual studio 2024. Tried updating to the latest visual studio code 1.96 and same error. Opened up the VSIX as a zip and couldn't find a package,json file.

**Comment by cwensley** [comment]
Hi @benjaminfortunato, thanks for trying it out!  However, the vsix is for Visual Studio 2022 for Windows only, not Visual Studio Code.  There is no Visual Studio 2024 as far as I'm aware.  For Visual Studio Code or command line, you can install the templates package via `dotnet new install Rhino.Templates.8.15.0 2.nupkg`

File: Rhino.Templates/content/CSGrasshopper2/.template.config/template.json [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        "RhinoVersion": {
+            "type": "parameter",
+            "description": "Version of Rhino",
+            "datatype": "choice",
+            "defaultValue": "8",
+            "choices": [
+                {
+                    "choice": "8",
+                    "description": "Version 8"
+                }
```
[the remark this answers is not in the document]
Comment (cwensley) [reply Rhino.Templates/content/CSGrasshopper2/.template.config/template.json:38]: Do you think I should add that to all of the templates, or just the GH2 template?

[the remark this answers is not in the document]
Comment (cwensley) [reply Rhino.Templates/content/CSGrasshopper2/.template.config/template.json:38]: I think I might add 9 WIP at some later date to all of the templates.

**Pull request merged**